addons/binding/org.openhab.binding.kostal.inverter/src/main/java/org/openhab/binding/kostal/inverter/ChannelConfig.java

```diff
@@ -0,0 +1,13 @@
+package org.openhab.binding.kostal.inverter;
```

Comment:
Could you pls run `mvn license:format` on the repo root in order to add the default license headers on all files?